policyDefinitions/App Configuration/app-configuration-stores-should-should-have-soft-delete-enabled-of-7-days/azurepolicy.json

```diff
@@ -0,0 +1,59 @@
+{
+  "name": "3557ee6d-ff74-49a7-8684-b0c83ce44bed",
+  "type": "Microsoft.Authorization/policyDefinitions",
+  "properties": {
+    "displayName": "App Configuration stores should should have soft delete enabled of 7 days",
+    "description": "Deleting a App Configuration store without soft delete enabled permanently deletes all data it contains",
+    "metadata": {
+      "version": "1.0.0",
+      "category": "App Configuration"
+    },
+    "mode": "All",
+    "parameters": {
+      "effect": {
+        "type": "String",
+        "metadata": {
+          "displayName": "Effect",
+          "description": "Deny, Audit or Disabled the execution of the Policy"
+        },
+        "allowedValues": [
+          "Deny",
+          "Audit",
+          "Disabled"
+        ],
+        "defaultValue": "Audit"
+      }
+    },
+    "policyRule": {
+      "if": {
+        "allOf": [
+          {
+            "field": "type",
+            "equals": "Microsoft.AppConfiguration/configurationStores"
+          },
+          {
+            "not": {
+              "field": "Microsoft.AppConfiguration/configurationStores/createMode",
+              "equals": "recover"
+            }
+          },
+          {
+            "anyOf": [
+              {
+                "field": "Microsoft.AppConfiguration/configurationStores/softDeleteRetentionInDays",
+                "exists": "false"
+              },
+              {
+                "field": "Microsoft.AppConfiguration/configurationStores/softDeleteRetentionInDays",
+                "notequals": 7
```

Comment:
can we change this value of 7 to be a parameter so that it can be modular for any community members?

Reply:
Hi @Joshua-Donovan,

I will take care of that once I'm back from holiday

Reply:
Hey @tdefise checking in on if you'd be able to make this change?